src/rpc/util.h

```diff
@@ -174,6 +198,7 @@ struct RPCArg {
           m_type_str{std::move(type_str)}
     {
         CHECK_NONFATAL(type != Type::ARR && type != Type::OBJ);
+        CheckFallback();
```

Comment:
Woudn't this be better checked in the RPCHelpMan constructor, which also checks the RPCArg name?

Reply:
:facepalm: RPCArg is a recursive datastructure, so it makes more sense to check this here (like you did initially)